string_utils/fms_string_utils_binding.c

```diff
@@ -184,3 +184,8 @@ int fms_find_unique(char** arr, int *n)
 
   return nfind;
 }
+
+char * cstring2cpointer (char * cs)
```

Comment:
I think this is my fault, but this needs to be documented.  
It returns the input pointer as a C string.